UnityProject/Assets/Scripts/Items/Weapons/Gun.cs

```diff
@@ -881,15 +881,7 @@ private Vector2 CalcProjectileDirections(Vector2 direction, int iteration)
 			//This is for shotgun spread and similar multi-projectile weapons
 			float angle = Mathf.Atan2(direction.y, direction.x) * Mathf.Rad2Deg;
 			float angleVariance = iteration / 1f;
-			float angleDeviation;
-			if (iteration % 2 == 0)
-			{
-				angleDeviation = angleVariance; //even
-			}
-			else
-			{
-				angleDeviation = -angleVariance; //odd
-			}
+			float angleDeviation = Convert.ToBoolean(iteration & 1) ? angleVariance : -angleVariance;
```

Comment:
```suggestion
			float angleDeviation = iteration % 2 == 0 ? angleVariance : -angleVariance;
```
if you wanted to oneline it.

Reply:
ugg, shame a bit check wont work as it would be cheaper